app/services/moab_storage_root_reporter.rb

```diff
@@ -36,21 +36,26 @@ def moab_detail_csv_list(errors_only: false)
         complete_moabs_and_preserved_objects_in_storage_root
       end
 
-    header_row = [
-      ['druid', 'previous storage root', 'current storage root', 'last checksum validation', 'last moab validation', 'status', 'status details']
-    ]
+    header_row = ['druid',
+                  'previous storage root',
+                  'current storage root',
+                  'last checksum validation',
+                  'last moab validation',
+                  'status',
+                  'status details']
 
-    # cols doesn't include storage_root name (available via storage_root ivar).  also we're not instantiating AR objects, so we have to translate
-    # the underlying status enum
+    # cols doesn't include storage_root name (available via storage_root ivar)
```

Comment:
meh - i'd leave this comment out.

Reply:
sure, see https://github.com/sul-dlss/preservation_catalog/pull/1374